### ISSUE TITLE
result-image-format should default to input format

### ISSUE THREAD

**Issue by hackel** [alexk111/ngImgCrop#107]
If I don't set an explicit result-image-format, then the format should remain unchanged.  (e.g. if I upload a jpeg, it should output as image/jpeg, png as image/png, etc.).  This could of course be set as an option for backward-compatibility.

I'm currently retrieving the mime-type via an additional HTTP HEAD request for the content-type, but this is obviously sub-optimal, and won't work for images that are not yet uploaded.

**Comment by vamsivarikuti** [comment]
Yes, `result-image-format` should default to input file format (OR) it should accept angular variable that specifies the format.

Otherwise all images will have same format, so whats the point in having this attribute?
